docs/data/routes/docs/nextjs/getting-started/walkthrough-dotnetnew/en.md

```diff
@@ -3,4 +3,193 @@ name: walkthrough-dotnetnew
+
+2. Check if you have the Internet Information Server running on port 443:
+
+   ```powershell
+   Get-Process -Id (Get-NetTCPConnection -LocalPort 443).OwningProcess
+   ```
+
+   If you do, you must stop it:
+
+   ```powershell
+   iisreset /stop
+   ```
+
+3. Check if you have Apache Solr or any other service running on port 8984:
+
+   ```powershell
+   Get-Process -Id (Get-NetTCPConnection -LocalPort 8984).OwningProcess
+   ```
+
+   If you do, you must stop it:
+
+   ```powershell
+   Stop-Service -Name "<the name of your service>"
+   ```
+
+   Or, if you have started it with Non-Sucking Service Manager, stop it with this command:
+
+   ```powershell
+   nssm stop "<the name of your service>"
+   ```
+
+4. Go to your solutions folder and create the `MyProject` solution and project:
+
+   ```powershell
+   dotnet new sitecore.nextjs.gettingstarted -n MyProject 
+   ```
+
+5. Go to the `MyProject` folder
+
+6. You use the provided `init.ps1` script to prepare the following items for the Sitecore container environment:
+
+   * A valid/trusted wildcard certificate for `*.myproject.localhost`.
+   * Hosts file entries for `myproject.localhost`.
+   * Required environment variable values in `.env` for the Sitecore instance.
+
+   > * Using non-Latin characters in the project/folder name can give unexpected results because of character limitations in Docker registry names and URLs.
+   >
+   > * See the [Sitecore Containers documentation](https://doc.sitecore.com/developers/101/developer-tools/en/containers-in-sitecore-development.html) for more information on these preparation steps.
+
+   Running the `init.ps1` script with the `-InitEnv` switch populates the `.env` variables required by the Sitecore container environment. It leverages `mkcert` to generate certificates.
+
+   To prepare the Sitecore container environment:
+
+   ```powershell
+   .\init.ps1 -InitEnv -LicenseXmlPath "<C:\path\to\license.xml>" -AdminPassword "<desired password>"
+   ```
+
+   > You must use an elevated/Administrator Windows PowerShell 5.1 prompt for this command. PowerShell 7 is not supported at this time.
+
+   Out of the box, this example does not include a reference to the `.env` file in the `.gitignore` file. This is so that developers can share initialized environment variables. If you check your `.env` file into source control, other developers can prepare a certificate and hosts file entries by simply running the `init.ps1` script.
+
+   If your Sitecore solution and/or its data are sensitive, we recommend that you keep the `.env` file excluded from source control and provide another means of centrally configuring the information within.
+
+   > If this is your first time using `mkcert` with Node.js, you must set the `NODE_EXTRA_CA_CERTS` environment variable in your user or system environment variables. For example: 
+   >
+   > ```powershell
+   > setx NODE_EXTRA_CA_CERTS C:\<path to user>\AppData\Local\mkcert\rootCA.pem
```

Comment:
Is the cert guaranteed to be at this path?

Reply:
`init.ps1` will prompt the user to set the variable. Re-worked this section to that effect.